### PR TITLE
chore: add numerous AVT tests

### PR DESCRIPTION
**Pull request by davidmenendez**
Contributes to #5066 #5069 #5070 & #5071 

also makes small adjustment to formatter command and pre-commit

### CODE REVIEW

File: packages/ibm-products/src/components/DecoratorBase/DecoratorBase.mdx [hunk 1]
```diff
@@ -1,11 +1,7 @@
...
-import {
-  getStoryId,
-  CodesandboxLink,
-} from '../../global/js/utils/story-helper';
-import { DecoratorBase } from '.';
-import * as DecoratorBaseStories from './DecoratorBase.stories';
+{/* cspell:ignore setlabeltitle */} import { Story, ArgsTable, Source, Canvas }
+from '@storybook/addon-docs'; import { getStoryId, CodesandboxLink, } from
+'../../global/js/utils/story-helper'; import { DecoratorBase } from '.';
+import \* as DecoratorBaseStories from './DecoratorBase.stories';
```
Comment (annawen1) [inline packages/ibm-products/src/components/DecoratorBase/DecoratorBase.mdx:4]: the formatting here is all bunched up and makes it harder to read the imports, is this intentional?

Comment (davidmenendez) [reply packages/ibm-products/src/components/DecoratorBase/DecoratorBase.mdx:4]: definitely not whoops

**Comment by davidmenendez** [comment]
@annawen1 fixt

**Review by annawen1 — APPROVED** [review]
LGTM!

**Pull request merged**